cmd/internal/flag.go

```diff
@@ -115,8 +115,8 @@ func initRegistryClientFlags() {
 
 func initImageVerifyCacheFlags() {
 	flag.BoolVar(&imageVerifyCacheEnabled, "imageVerifyCacheEnabled", true, "Whether to use a TTL cache for storing verified images.")
-	flag.Int64Var(&imageVerifyCacheMaxSize, "imageVerifyCacheMaxSize", 1000, "Max size limit for the TTL cache, 0 means default 1000 size limit.")
-	flag.DurationVar(&imageVerifyCacheTTLDuration, "imageVerifyCacheTTLDuration", 60*time.Minute, "Max TTL value for a cache, 0 means default 1 hour TTL.")
+	flag.Int64Var(&imageVerifyCacheMaxSize, "imageVerifyCacheMaxSize", 1000, "Maximum number of keys that can be stored in the TTL cache, The default 1000 size limit, 0 sets the value to default.")
+	flag.DurationVar(&imageVerifyCacheTTLDuration, "imageVerifyCacheTTLDuration", 60*time.Minute, "Maximum TTL value for a cache, default is 1 hour, 0 sets the value to default.")
```

Comment:
```suggestion
	flag.DurationVar(&imageVerifyCacheTTLDuration, "imageVerifyCacheTTLDuration", 60*time.Minute, "Maximum TTL value for a cache in minutes. Default is 60. 0 sets the value to default.")
```

Reply:
Its not in minutes, it takes duration as input, so `1h`, `2m30s` etc.

Reply:
So what's the default? `1h` or `60m`?

Reply:
Its a duration value, `60m`

Reply:
Ok, check the suggestion now.

Reply:
Yep, looks good to me!